README.md

```diff
@@ -1,15 +1,57 @@
 # ILP Plugin XRP Escrow
 > ILP ledger plugin using the escrow features of ripple
 
-**NOTE**: This plugin is under development, and should not yet be used in
-production or with funds on the live network.
+**NOTE**: This plugin is under development, and should not yet be used
+with funds of over ~ 0.01 USD in value.
+
+## Step 1: create a Ripple wallet.
+
+This can easily be done with the following node script:
+```js
+var keypairs = require('ripple-keypairs')
+
+const seed = keypairs.generateSeed()
+const keypair = keypairs.deriveKeypair(seed)
+const address = keypairs.deriveAddress(keypair.publicKey)
+console.log({
+  secret: seed,
+  account: address,
+  keypair
+})
+```
+
+Make sure your store your secret (seed) in a safe place and then use your account (address) 
```

Comment:
I think we need a stronger warning here, because if you ever lose your seed then all of your money is irretrievably lost.